understory_box_tree/src/tree.rs

```diff
@@ -59,6 +59,8 @@ pub struct QueryFilter {
     pub visible_only: bool,
     /// If true, only consider nodes marked [`NodeFlags::PICKABLE`] (hit-test).
     pub pickable_only: bool,
+    /// If true, only consider nodes marked [`NodeFlags::FOCUSABLE`].
+    pub focusable_only: bool,
```

Comment:
Starting to think we should switch to a bit field and have helper methods / builder style ...

Reply:
But maybe in a followup.